PORTAL/jobs/_pyperformance.py

```diff
@@ -210,9 +210,9 @@ def load(
     def _load(self):
         for suite in self.SUITES:
             try:
-                names = self._cache[suite]
+                self._cache[suite]
             except KeyError:
-                names = self._cache[suite] = self._load_suite(suite)
+                self._cache[suite] = self._load_suite(suite)
```

Comment:
I'm not sure what `names` was used for before, but if we're dropping it then we can simplify here:

```python
            if suite not in self._cache:
                self._cache[suite] = self._load_suite(suite)
```